api/v1alpha1/nnf_node_storage_types.go

```diff
@@ -83,7 +65,7 @@ type LustreStorageSpec struct {
 	FileSystemName string `json:"fileSystemName,omitempty"`
 
 	// TargetType is the type of Lustre target to be created.
-	// +kubebuilder:validation:Enum=MGT;MDT;MGTMDT;OST
+	// +kubebuilder:validation:Enum=mgt;mdt;mgtmdt;ost
```

Comment:
What is the reason for switching the case here?  I'm concerned about this causing breakage in other areas.  Does Flux need to know about this change?

Also, there are still uppercase references in `config/samples`.

Reply:
We were converting from lowercase in the directivebreakdown/servers resource where they're used as labels to uppercase for the NnfStorage/NnfNodeStorage. We were doing that because nnf-ec wanted them as upper case for its arguments. We don't need them as uppercase anymore with nnf-ec out of the picture for Lustre, so I changed it to keep the case consistent in all our resources.

Flux shouldn't be looking at the NnfStorage or NnfNodeStorage, and it shouldn't be aware of Lustre targets at all. The directivebreakdown and servers resources haven't changed in this area.

I'll update the samples.